core/src/main/java/io/github/tcdl/msb/impl/ResponderServerImpl.java

```diff
@@ -60,9 +64,9 @@ public ResponderServer listen() {
     void onResponder(ResponderImpl responder) {
         Message originalMessage = responder.getOriginalMessage();
         Object rawPayload = originalMessage.getRawPayload();
-        T request = payloadMapper.convertValue(rawPayload, payloadTypeReference);
-        LOG.debug("[{}] Process message with id: [{}]", namespace, originalMessage.getId());
         try {
+            T request = Utils.convert(rawPayload, payloadTypeReference, payloadMapper);
+            LOG.debug("[{}] Process message with id: [{}]", namespace, originalMessage.getId());
```

Comment:
Is it correct to send 500 Internal Error?Are we agreed on it with our architects?From my side in this case we should use 400 Bad Request)